vite.config.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="vitest" />
+/// <reference types="@vitest/browser/matchers" />
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig({
+  plugins: [react(), tsconfigPaths()],
+  server: {
+    allowedHosts: [
+      "5173-wpilibsuite-systemcoreb-r536zqlysb3.ws-us117.gitpod.io",
```

Comment:
How can this be made generic?

Reply:
Oh, sorry, that's left over from my development environment. Removing right now.